src/config.js

```diff
@@ -70,6 +69,17 @@ export function getConfig(options) {
   const transformingVisitors = require(lintConfig.transformers);
   lintConfig.transformingVisitors = transformingVisitors;
 
+  if (!resolvedConfig.lint) {
+    resolvedConfig.lint = {};
+  }
+
+  resolvedConfig.lint.headers = (resolvedConfig.lint && resolvedConfig.lint.headers) ? (resolvedConfig.lint.headers.map((header) => ({
```

Comment:
`(resolvedConfig.lint && resolvedConfig.lint.headers || []).map` instead of ternary operator would be less verbose